src/collective/sitter/browser/templates/sitteraccount.pt

```diff
@@ -50,13 +50,13 @@
         <li tal:condition="python: view.sitter_state.has_accepted() and not view.sitter_state.is_created()">
           <a class="context" href="${view/add_entry_url}" i18n:translate="create_entry">Eintrag erstellen</a>
         </li>
-        <li tal:condition="python: view.sitter_state.is_created() and not view.sitter_state.is_submitted()">
+        <li tal:condition="python: view.sitter_state.has_accepted() and view.sitter_state.is_created() and not view.sitter_state.is_submitted()">
           <a class="context" href="${view/submit_entry_url}" i18n:translate="submit_entry">Eintrag einreichen</a>
         </li>
-        <li tal:condition="python: view.sitter_state.is_created() and not view.sitter_state.is_deleted()">
+        <li tal:condition="python: view.sitter_state.has_accepted() and view.sitter_state.is_created() and not view.sitter_state.is_deleted()">
           <a class="context" href="${view/edit_entry_url}" i18n:translate="edit_entry">Eintrag bearbeiten</a>
         </li>
-        <li tal:condition="python: view.sitter_state.is_created() and not view.sitter_state.is_deleted()">
+        <li tal:condition="python: view.sitter_state.has_accepted() and view.sitter_state.is_created() and not view.sitter_state.is_deleted()">
```

Comment:
Zumindest das Löschen sollten wir erlauben, ohne daß man den Bedingungen zugestimmt hat – immerhin wäre das ja ein Grund, warum man löschen möchte. Aber eigentlich sollte is_created ja has_accepted implizieren, da es keinen expliziten Widerruf gibt, es sei denn, wir wollen dem korrekten Ablauf nicht trauen müssen. Persönlich finde ich die Logik gut, has_accepted noch mal zu prüfen, aber evtl. möchte man für diese komplexe Abfrage eine neue semantische Methode auf dem SitterState.

Reply:
> Zumindest das Löschen sollten wir erlauben, ohne daß man den Bedingungen zugestimmt hat – immerhin wäre das ja ein Grund, warum man löschen möchte. 

Aber ohne Zustimmung hätte man ja keinen Eintrag anlegen können, der dann gelöscht werden soll. Und nur Zustimmung zurückziehen geht ja nicht.

Reply:
Das stimmt, und es ist in etwas anderer Formulierung ein Teil dessen, was ich vorher schon ausdrücken wollte. ;-) Also den aktuell implementierten Ablauf vorausgesetzt, kommt es aufs Gleiche raus, ob wir has_accepted noch mal prüfen oder nicht. Von daher ist es für mich jetzt auch OK, das zu mergen, aber eigentlich finde ich es komisch, daß da im Prinzip steht: "Falls ein Eintrag existiert, obwohl die Zustimmung zu den AGB aus irgendwelchen Gründen nicht gegeben ist, dann soll er nicht gelöscht werden können."